assets/js/base/components/label/index.tsx

```diff
@@ -1,27 +1,29 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
-import { Fragment } from 'react';
+import { Fragment } from '@wordpress/element';
 import classNames from 'classnames';
+import type { HTMLAttributes } from 'react';
```

Comment:
Importing this as a type from `react` because:

- `HTMLAttributes` is not exported from `@types/wordpress__element`.
- We don't have anything other than the interface imported here.